subset/security/security_passwords/src/main/resources/defaultPasswords.json

```diff
@@ -0,0 +1,78 @@
+{
+  "Axis2":{
+    "Usernames":"admin,root",
+    "Passwords":"axis2,1988"
+  },
+  "DB2":{
+    "Usernames":"admin,dasusr1,db2admin,db2fenc1,db2inst1",
+    "Passwords":"admin,dasusr1,db2admin,db2fenc1,db2inst1,db2pass,db2password,db2pw"
+  },
+  "FTP":{
+    "Usernames":"admin,anonymous,ftp,ftp_admi,ftp_inst,ftp_nmc,ftp_oper,ftpuser,login,rapport,root,user,xbox",
+    "Passwords":"1234,access,chrome@example.com,Exabyte,ftp,help1954,IEUser@,kilo1987,mozilla@example.com,pass,password,pbxk1064,r@p8p0r,tuxalize,xbox"
+  },
+  "HTTP":{
+    "Usernames":"admin,apc,axis2,cisco,connect,manager,newuser,pass,private,root,security,sitecom,sys,system,tomcat,user,wampp,xampp,xampp-dav-unsecure",
+    "Passwords":"1234,admin,apc,cisco,connect,default,letmein,manager,none,pass,password,ppmax2011,root,sanfran,security,sitecom,sys,system,tomcat,turnkey,user,wampp,xampp"
+  },
+  "MSSQL":{
+    "Usernames":"Administrator,ARAdmin,entldbdbo,entldbreader,mon_user,probe,repl_publisher,repl_subscriber,sa,WinCCConnect",
+    "Passwords":"2WSXcder,AR#Admin#,blank,dbopswd,pass,pass1,password,rdrpswd"
+  },
+  "MySQL":{
+    "Usernames":"admin,mysql,root",
+    "Passwords":"blank,pass,pass1,password,vicidia1now"
+  },
+  "PostgreSQL":{
+    "Usernames":"admin,postgres,scott,tom",
+    "Passwords":"admin,password,postgres,tiger"
+  },
+  "SMB":{
+    "Usernames":"backuphelpdesk",
+    "Passwords":"backuphpinvent"
+  },
+  "SNMP":{
+    "Usernames":"<BLANK>",
+    "Passwords":"0392a0,1234,2read,4changes,access.adm,Admin,admin,agent,agent_steal,all,allprivate,allpublic,ANYCOM,apc,bintec,blue,c,C0de,cable-d,canon_admin,cc,CISCO,cisco,community,core,CR52401,debug,default,dilbert,enable,field,field-service,freekevin,fubar,guest,hello,hp_admin,IBM,ibm,ILMI,ilmi,Intermec,intermec,internal,l2,l3,manager,mngt,monitor,netman,network,NoGaH$@!,none,openview,OrigEquipMfr,pass,password,pr1v4t3,private,PRIVATE,Private,proxy,publ1c,public,PUBLIC,Public,read,read-only,read-write,readwrite,red,regional,rmon,rmon_admin,ro,root,router,rw,rwa,s!a@m#n$p%c,san-fran,sanfran,scotty,SECRET,Secret,secret,SECURITY,Security,security,seri,SNMP,snmp,SNMP_trap,snmpd,snmptrap,solaris,SUN,sun,superuser,SWITCH,Switch,switch,SYSTEM,System,system,tech,TENmanUFactOryPOWER,TEST,test,test2,tiv0li,tivoli,trap,world,write,xyzzy,yellow"

```

Comment:
Are these something we control?  If so, or in the future, this would be much better encoded as an actual JSON list... Having the CSV here is somewhat of a nightmare to maintain.

Reply:
JSON file has been formatted to resemble a proper JSON list and the Java has been changed to read from this new format